frontend/packages/ux-editor/src/components/Properties/Properties.tsx

```diff
@@ -24,72 +24,78 @@ export const Properties = () => {
     }
   };
 
+  if (!formItem) {
+    return (
+      <div className={classes.root} key={formItemId}>
+        <PageConfigPanel />
+      </div>
+    );
+  }
```

Comment:
Very good! Super nitpicking, but could you move this block of code above the `toggleOpen` function as I mentioned in my initial feedback? 🙂 I think it would improve the structure, since `toggleOpen` will never be called if `formItem` is not defined.